astro/src/content/json/themes/templates.json

```diff
@@ -1206,6 +1206,80 @@
+      },
+      {
+        "name": "phoneNumber",
+        "type": "String",
+        "description": "The current value of the user's phone number. This may be useful to indicate to the user which phone number was sent a verification code."
+      },
+      {
+        "name": "showCaptcha",
+        "type": "Boolean",
+        "description": "A boolean that controls whether or not to include captcha scripts and show the captcha challenge (or message for invisible captcha).",
+        "since": "1.30.0"
+      },
+      {
+        "name": "verificationId",
+        "type": "String",
+        "description": "The verification Id that was included on as a URL parameter. This is the high entropy value that will be paired with the low entropy one time code to complete phone number verification."
+      }
+    ]
+  },
+  {
+    "displayName": "Phone number verification complete",
+    "fieldName": "phoneComplete",
+    "description": "This page is used after a user has verified their phone number by clicking the URL in the message. After FusionAuth has updated their user object to indicate that their phone number was verified, the browser is redirected to this page.",
+    "path": "/phone/complete"
+  },
+
+  {
+    "displayName": "Phone number verification re-sent",
+    "fieldName": "phoneSent",
+    "description": "This page is used after a user has asked for the verification message to be resent. This can happen if the URL in the message expired and the user clicked it. In this case, the user can provide their phone number again and FusionAuth will resend the message. After the user submits their phone number and FusionAuth re-sends a verification message to them, the browser is redirected to this page.",
+    "path": "/phone/sent",
+    "variables": [
+      {
+        "name": "phoneNumber",
+        "type": "String",
+        "description": "The phone number that requested to receive a new verification request message."
+      }
+    ]
+  },
+  {
+    "displayName": "Phone number verification",
+    "fieldName": "phoneVerify",
+    "description": "This page is rendered when a user clicks the URL from the verification message and the `verificationId` has expired. FusionAuth expires `verificationId` after a period of time (which is configurable). If the user has a URL from the verification message that has expired, this page will be rendered and the error will be displayed to the user.",
+    "path": "/phone/verify",
+    "variables": [
+      {
+        "name": "postMethod",
+        "type": "Boolean",
+        "description": "Defaults to `true` on this page."
+      },
+      {
+        "name": "showCaptcha",
+        "type": "Boolean",
+        "description": "A boolean that controls whether or not to include captcha scripts and show the captcha challenge (or message for invisible captcha).",
+        "since": "1.30.0"
```

Comment:
I think remove this because the field has been available in this template since the beginning.

Reply:
fixed